src/LaunchDarkly.Client/Operator.cs

```diff
@@ -158,9 +123,26 @@ public static bool Apply(string op, JValue uValue, JValue cValue)
             return false;
         }
 
+        private static bool TryCompareNumericValues(JValue x, JValue y, out int result)
+        {
+            if (!IsNumericValue(x) || !IsNumericValue(y))
+            {
+                result = default(int);
+                return false;
+            }
+
+            result = x.CompareTo(y);
```

Comment:
Comparison logic between different numeric value types is already implemented by Newtonsoft:

https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/Linq/JValue.cs#L244